.env.example

```diff
@@ -0,0 +1,20 @@
+ACCOUNT1=0xc5e8f61d1ab959b397eecc0a37a6517b8e67a0e7cf1f4bce5591f3ed80199122
```

Comment:
why do you push private keys (tests) to github haha I think the main purpose is to put it in some ignore files
also 4-5 accounts should be sufficient
should put the test wallets file to gitignore, each of us will have our own file locally

Reply:
oh my bad, I forgot to delete them from `.env.example`.
I purpose to make a template for any new member to run test in a familiar way after pulling source code without asking anyone.